nhd/NHDCommon.py

```diff
@@ -15,6 +15,8 @@ class NHDCommon:
                     'ERROR': 'bold_red',
                     'CRITICAL': 'bold_red' }             
 
+    NHD_MAINT_LABEL = 'sigproc.viasat.io/maintenance'
```

Comment:
This seems out of place in NHD - shouldn't this be created by the MKO playbook when creating the cluster nodes?